js/components/controlBar.js

```diff
@@ -196,13 +186,17 @@ var ControlBar = React.createClass({
 
   volumeHighlight:function() {
```

Comment:
We still want to use the opacity/color settings from the skin config for the icon, so I think this part needs to stay in the JS.